client/src/components/UtcDate.vue

```diff
@@ -12,13 +11,12 @@ const props = withDefaults(defineProps<UtcDateProps>(), {
     mode: "date",
 });
 
-// Component assumes ISO format date, but note that in Galaxy this won't have
-// TZinfo -- it will always be Zulu
 const parsedDate = computed(() => parseISO(`${props.date}Z`));
 const elapsedTime = computed(() => formatDistanceToNow(parsedDate.value, { addSuffix: true }));
 const fullISO = computed(() => parsedDate.value.toISOString());
-const pretty = computed(() => `${formatInTimeZone(parsedDate.value, "Etc/Zulu", "eeee MMM do H:mm:ss yyyy")} UTC`);
+const pretty = computed(() => `${format(parsedDate.value, "eeee MMM do H:mm:ss yyyy zz")}`);
```

Comment:
What about using the combination of long localized date and time?

```suggestion
const pretty = computed(() => `${format(parsedDate.value, "PPPPpppp")}`);
```

Reply:
Both options yield the same outcome, but I prefer the current one because it is easier to read.

Reply:
@itisAliRH Sorry I missed your reply since I wasn't tagged and didn't notice you were replying to my old comment.

The two patterns don't yield the same outcome, see the following examples:

```js
import { format } from "date-fns";
import { it } from "date-fns/locale";

format(new Date(2014, 0, 11), 'eeee MMM do H:mm:ss yyyy zz')  // Saturday Jan 11th 0:00:00 2014 GMT+0
format(new Date(2014, 0, 11), 'PPPPpppp')  // Saturday, January 11th, 2014 at 12:00:00 AM GMT+00:00
format(new Date(2014, 0, 11), 'eeee MMM do H:mm:ss yyyy zz', { locale: it })  // sabato gen 11 0:00:00 2014 GMT+0
format(new Date(2014, 0, 11), 'PPPPpppp', { locale: it })  // sabato 11 gennaio 2014 00:00:00 GMT+00:00
```

from which we can see that 'PPPPpppp' has the following advantages:
- The year is kept part of the date and not put between the time and timezone
- The time is localised too.
- The order of day of the month and month is localised (in Europe we write the day before the month)

Reply:
Sorry I missed this conversation hadn't resolved.  What about `PPPppp`?

This seems even better to me as a natural display of a timestamp I'd expect it an application.

```
Sunday, July 2nd, 2023 at 2:44:16 PM GMT-04:00 
vs 
July 2nd, 2023 at 2:44:16 PM GMT-4
```

Reply:
Not a strong opinion, but sometimes it's convenient to know straightaway the day of the week (e.g. of when you started a job) without having to look at the calendar.
On the other hand screen space is precious, so...

Reply:
@nsoranzo Yeah, I thought about that a bit and to me it seems like day of the week is relevant for only a brief period of time (<1w).  I guess we could display day if recent, otherwise not, but I think I lean towards consistent brevity?  Only a slight preference, though, if anyone feels strongly I'm fine with whatever :)